framework/test/jest/unit/specs/modules/chain/dpos/apply.spec.js

```diff
@@ -99,6 +113,25 @@ describe('dpos.apply()', () => {
 
 			expect(stubs.storage.entities.Account.update).not.toHaveBeenCalled();
 		});
+
+		it('should call RoundDelegates.delete method when applying genesis block', async () => {
```

Comment:
Maybe change the description and make sure `create` is called after `delete`?

```suggestion
		it('should save round 1 active delegates list in RoundDelegates entity after applying genesis block', async () => {
```

Reply:
This functionality is part of: https://www.npmjs.com/package/jest-extended#tohavebeencalledbefore Need to check if we can include this.

Reply:
Nvm, this is already present in SDK.